.github/workflows/managed.yml

```diff
@@ -55,6 +55,9 @@ jobs:
           path: ~/go/pkg/mod
           key: ${{ runner.os }}-go-modules-${{ hashFiles('**/go.sum') }}
           restore-keys: ${{ runner.os }}-go-modules-
+          
+      - name: Mark the root directory of pmm as safe
```

Comment:
It should go after `make env-compose-up`, not before.

Reply:
oh, you are right. fixed